docs/modules.md

```diff
@@ -27,7 +28,7 @@
 
 #### Defined in
 
-[types/responses.ts:147](https://github.com/GigaHierz/toucan-sdk/blob/b837065/src/types/responses.ts#L147)
+[types/responses.ts:155](https://github.com/0xmichalis/toucan-sdk/blob/1488e66/src/types/responses.ts#L155)
```

Comment:
Why not `ToucanProtocol/toucan-sdk`?

Reply:
uhh, this is how `yarn docs` works apparently. I'll raise a ticket to address this as a follow-up.

Reply:
Tracking in https://linear.app/toucan/issue/LILA-7809